src/main/webapp/resources/js/utilities/validation-utilities.js

```diff
@@ -0,0 +1,10 @@
+const emailRegex = /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/;
+
+/**
+ * Ensure that an email address is formatted correctly.
+ * This regular expression is taken from (https://stackoverflow.com/questions/46155/how-to-validate-an-email-address-in-javascript)
+ * @param {string} email address to validate
+ * @returns {boolean}
+ */
+export const validateEmail = email =>
```

Comment:
Is there no existing function or library to validate emails in JavaScript? I'm surprised we haven't done this before (since we deal with email addresses a lot).

Reply:
Okay, sounds good. Thanks.